content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -311,23 +311,17 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).
-
 For Cilium to work correctly, you must pass `--pod-network-cidr=10.217.0.0/16` to `kubeadm init`.
 
-These commands will deploy Cilium with its own etcd managed by etcd operator.
-
-_Note_: If you are running kubeadm in a single node please untaint it so that
-etcd-operator pods can be scheduled in the control-plane node.
-
-```shell
-kubectl taint nodes <node-name> node-role.kubernetes.io/master:NoSchedule-
-```
+Cilium can provide kube-proxy functionality. To disable kube-proxy, pass
+`--skip-phases=addon/kube-proxy` to `kubeadm init`. Afterwards, you will need
+to configure Cilium to enable the BPF NodePort feature and to specify the kube-apiserver
+address, see [Kubernetes without kube-proxy](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/).
```

Comment:
@brb 
similar comment as https://github.com/kubernetes/website/pull/16313#discussion_r324172413

i know this was closed, but i'm going to hold on this change:
/hold

we should not recommend skipping kube-proxy unless the users want to get stranded during upgrades.